src/components/FilterSection.tsx

```diff
@@ -9,6 +14,10 @@ const FilterSection: React.FC = () => {
         Data table filters.
       </h3>
       <Grid container spacing={3}>
+        <Grid size={4} spacing={1}>
+          <h3>By Name:</h3>
```

Comment:
replace this h3 tag with typography `/shared`. to maintain consistency of the project